app/src/main/res/values/themes.xml

```diff
@@ -22,7 +22,6 @@
         <item name="colorPrimaryVariant">@color/teal_900</item>
         <item name="colorOnPrimary">@color/white</item>
         <!-- Secondary brand color. -->
-        <!-- Unused atm -->
```

Comment:
double check if the starter code doesn;t have this comment

Reply:
This was built off of `activities` branch and wasn't change. Not in the `starter` branch either.